index.html

```diff
@@ -0,0 +1,178 @@
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+            <div
+              class="block blocke my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/forum.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/ignite.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">HTML/CSS Editors</p>
+              <p class="program-description text-white">
+                Tools for editing HTML/CSS code, making web design easier.
+                Examples: Sublime Text, Atom, Brackets, Dreamweaver, Notepad++.
+              </p>
+            </div>
+          </div>
+
+          <button class="join p-4 mt-5">Join WebDev Bootcamp 2023</button>
+          <p class="desktop-buttone">
+            <a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+          </p>
+        </div>
+      </article>
+      <artical class="speakers mt-5 px-4 py-5 bg-white">
+        <h3 class="main-program text-center text-black">Featured Speakers</h3>
+        <h3 class="baseline text-center text-white fw-bold"></h3>
+        <img src="" alt="" />
+        <div class="main-speakers my-5"></div>
+
+        <button
+          id="seemore"
+          class="morex w-100 py-2 d-flex d-lg-none justify-content-center align-items-center"
+        >
+          More
+          <img class="arrow ms-2" src="icons/arrowdown.svg" alt="" />
+        </button>
+      </artical>
+      <article class="card partners mt-5 bg-dark text-white">
+        <h3 class="partner text-center mt-5">Partner</h3>
+        <h3 class="baseline fw-bold"></h3>
+        <div class="companies">
+          <div class="row d-flex justify-content-center mb-3">
+            <img class="col-4 py-4" src="icons/mozilla.svg" alt="" />
+            <img class="col-4" src="icons/google.svg" alt="" />
+            <img class="col-4" src="icons/naver.svg" alt="" />
+          </div>
+          <div class="row d-flex justify-content-center pb-4">
+            <img class="col-6" src="icons/daumkakao.svg" alt="" />
+            <img class="col-6" src="icons/airbnb.svg" alt="" />
+          </div>
+        </div>
+      </article>
+      <article>
```

Comment:
- In order to stick to Cindy's design, I suggest making sure the mobile version of the home page ends at the show more button.
- Let's make fix the button's border too to avoid any bugs

Your design:
![Screenshot from 2023-03-18 08-00-45](https://user-images.githubusercontent.com/97834160/226088316-deacb866-fc1f-46b3-b55e-d65fa545735b.png)

Cindy's:
![Screenshot from 2023-03-18 08-09-00](https://user-images.githubusercontent.com/97834160/226088348-9fd20a1d-6587-446f-a137-05a1100f0d17.png)

Reply:
Done

Reply:
I guess we are allowed to make some changes as mentioned in the specifications.

- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.